incubator/nextcloud/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+- name: mariadb
+  version: 0.7.0
```

Comment:
The version supports ranges. What about `~0.7.0` so that it can pull in patch level differences?